### ISSUE TITLE
[TCOTD2] Blood Thamuz is far less dangerous than the Doom one

### ISSUE THREAD

**Issue by Blue-Shadow** [Realm667/Re-Releases#268]
Apart from that invulnerability spell which they both share, Doom Thamuz has:
+ An explosive, albeit slow, homing attack, which does massive damage.
+ A three-volley attack where it fires napalm launcher projectiles (these are explosive, very fast and can insta-kill the player no matter how much health and armor he or she has).
+ Similar attack to the above, except it fires moderately fast, explosive, ripping, bouncing fireballs that do loads of damage.

Blood Thamuz has:
+ An attack where it summons slow-moving tentacles, that in a way, act as a meat shield and/or distraction.
+ A barrage of slow projectiles with good damage, but not to the point where you get killed instantly if you get hit by one of them (unless you're unhealthy and have no armor).

As you can see there, Doom Thamuz is the master of, pretty much, insta-killing, while Blood Thamuz can only dream of that power.

One thing I'm thinking is to give Blood Thamuz the ability to do leading shots with its barrage attack, much like what I gave the Shadow, in order to make dodging more tricky and prevent circle-strafing the boss to death. But that's all I've got.

**Comment by nashmuhandes** [comment]
Yeah the shot-loading of the Shadow is actually one thing I love about them. I think even just that would already increase the difficulty of the boss.

**Comment by Tormentor667** [comment]
I agree with that addition, that should help making both of them a challenge.

**Comment by nashmuhandes** [comment]
Amazing! Great job on cleaning up that spaghetti... :D

**Comment by Blue-Shadow** [comment]
Yeah, it was an annoying endeavor, to say the least. Hopefully, I'll be done with the tweaks tomorrow.

**Comment by nashmuhandes** [comment]
You deserve a cookie for that. I saw that stuff and also had ambitions to clean it up but it got too overwhelming... it's never a good experience to clean up spaghetti that I didn't write myself...

**Comment by Tormentor667** [comment]
Guys, this is going to be awesome :)
